src/components/WishList/WishListCard.js

```diff
@@ -0,0 +1,37 @@
+import React from "react";
+import { useWishlist } from "../../contexts/wishlist-context";
+
+function WishListCard({ product }) {
+  const { _id, name, price, img } = product;
+
+  const { wishDispatch } = useWishlist();
+  return (
+    <>
+      <div className="wishlist-card" id={_id}>
+        <div className="container-img">
+          <img className="wishlist-img" src={img} alt="product" />
```

Comment:
`alt="product"` is more generic, use name as alt here `alt={name}`

Reply:
done!